client/src/components/PledgItem/PledgeItem.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './style.css'
+const PledgeItem = props => {
+  const { element, history } = props
+  return (
+    <div className="pledge-div">
+      <img onClick={() => { console.log("history.push('')") }} className="pledge-div__img" src={element.img} />
```

Comment:
`img` elements must have an `alt` prop, either with meaningful text, or an empty string for decorative images

Reply:
@yaqootturman

Reply:
did it , check it again ,pleasr